pkg/ifaces/informer.go

```diff
@@ -38,6 +39,7 @@ type Event struct {
 type Interface struct {
 	Name   string
 	Index  int
+	MAC    [6]uint8
```

Comment:
@jotak last time I added field there nsString performance degraded, can pls check perf impact with this change to be sure ?

Reply:
sure, @memodi triggered a test but it failed, let me retry...

Reply:
ouch, bad news, there's +77% CPU in DNH :-/
... going to investigate that ... I'm surprised because doing just 2 maps lookup in the worst case didn't seem bad to me ; or maybe that's the fallback code invoked more than I thought

Reply:
I've found the problem I think. Problem is with OVN as it makes up MAC based on IPs in eth header, even when the if_index is 2 (eth0). So the flows MAC don't correspond in that case to the eth0 interface MAC, which triggers a cache miss, and fallback to syscall, which is why the CPU increases.
I've added an OVN specific optimization to avoid these syscalls in that situation, see last commit